### ISSUE TITLE
SA1027 does not detect tabs in xml documentation

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#1830]
Repro:
```csharp
///[Tab]Foo
class Program { }
```

This should report SA1027. It does not at the moment.

**Comment by sharwell** [comment]
How far do we want to go with this?

```csharp
/// <summary {tab}><cref{tab}name="{tab}Program"{tab}/></summary{tab}>
class Program { }
```
